src/styles/header.scss

```
@@ -0,0 +1,75 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  border-bottom: 1px solid #f3f3f3;
```

create variables for your colors
```suggestion
  border-bottom: 1px solid $c-....;
```